tools/uswds-color-tokens-figma.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: USWDS Color Token Template 
+nav: Tools
+permalink: /tools/uswds-color-token-template/
+sidenav: tools-side-nav
+---
+# {{ page.title }}
+
+<div class="iframe-container">
+<iframe class="responsive-iframe" style="border: 1px solid rgba(0, 0, 0, 0.1);" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FSm2zi00hmcM5VE4ZWeTulB%2FUSWDS-Color-Token-Template%3Fnode-id%3D0%253A1" allowfullscreen></iframe>
+</div>
+
+<a href="https://www.figma.com/community/file/1024394345443465950" class="usa-button usa-button--outline">Duplicate this file</a>
+
+## Getting Started
+This template can be used as a starting point for adopting the use of color tokens in your design systems in Figma. 
+
+In this file you’ll find the [theme color tokens](https://designsystem.digital.gov/design-tokens/color/theme-tokens/) and [state color tokens](https://designsystem.digital.gov/design-tokens/color/state-tokens/) as defined by USWDS. 
+
+The tokens are customizable for your projects, so use this file as a base to help you organize and name your tokens to align with USWDS guidance. 
+
+Each color token is saved as a color style. Each color style is named by the theme or state color token associated with it and includes a description that names the [system token](https://designsystem.digital.gov/design-tokens/color/system-tokens/) and the [settings](https://designsystem.digital.gov/documentation/settings/) variable that it is referencing. 
+
+## How to use this template
```

Comment:
I'm not sure how much value this is providing on this page. I would either think about how to do a high-level version of the instructions, or just leave it out.